compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -494,7 +494,7 @@ class OMR_EXTENSIBLE CodeGenerator
    //
    bool supports32bitAiadd() {return true;}  // no virt, default
    bool supportsMarshallingUnmarshallingIntrinsics() {return true;} // no virt, default
-   bool supportsMergingOfHCRGuards() {return false;} // no virt, default
+   bool supportsMergingGuards() {return false;} // no virt, default
```

Comment:
@mgaudet are these "no virt, default" comments still useful? This should be obvious given we are in the base OMRCodeGenerator.